pandas/io/json/json.py

```diff
@@ -473,7 +478,7 @@ def read_json(
 
     Returns
     -------
-    result : Series or DataFrame, depending on the value of `typ`.
+    Series or DataFrame, depending on the value of `typ`.
 
     See Also
     --------
```

Comment:
can you add a short description to the `See Also` item?

Reply:
You got it. And since read_json can return a Series depending on the `typ` parameter, I'll add in Series.to_json for good measure.